k8s/contiv-vpp-arm64.yaml

```diff
@@ -22,7 +22,8 @@ metadata:
   namespace: kube-system
 data:
   contiv.conf: |-
-    useNoOverlay: false
```

Comment:
Please do not change defaults in this file. Leave SRv6 disbaled by default for now. After that, we will be able to run regression tests (IPv4) on your merge request.

Reply:
fixed in revert commit https://github.com/contiv/vpp/pull/1523/commits/89026877015c3b75767871284bc6960e939eac69